flooder/src/main.rs

```diff
@@ -45,6 +41,12 @@ fn main() -> Result<(), anyhow::Error> {
             None => panic!("Needs --phrase or --seed"),
         },
     };
+
+    let rate_limiting = match (config.transactions_in_interval, config.interval_secs) {
```

Comment:
Output something for the user. `Config.rs` doesn't provide any info on how to use these parameters together.

Reply:
done